BlazorServer/run.bat

```diff
@@ -1,6 +1,6 @@
 start "C:\Program Files (x86)\Google\Chrome\Application\chrome.exe" "http://localhost:5000"
-d:
-cd D:\GitHub\WowPixelBot\BlazorServer
+c:
+cd C:\Users\Alexey\Desktop\Projects\WowClassicGrindBot\BlazorServer
```

Comment:
revert this change it has no relation to the hunter class

Reply:
I think it should be in gitignore

Reply:
Yea completely right about this 👍